contracts/modules/cover/Cover.sol

```diff
@@ -126,7 +126,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     uint segmentId;
 
     AllocationRequest memory allocationRequest;
-
+    uint maxUint = type(uint).max;
```

Comment:
This needlessly occupies a stack slot as we don't actually need this variable. Remove the variable and inline the `type(uint).max` expression.